apps/dokploy/templates/huly/docker-compose.yml

```diff
@@ -0,0 +1,176 @@
+      - MODEL_ENABLED=*
+      - ACCOUNTS_URL=http://${SERVER_ADDRESS}:3001
+      - ACCOUNT_PORT=3001
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  front:
+    image: hardcoreeng/front:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+      - elastic
+      - collaborator
+      - transactor
+    ports:
+      - 8087:8080
+    environment:
+      - SERVER_PORT=8080
+      - SERVER_SECRET=${SERVER_SECRET}
+      - ACCOUNTS_URL=http://${SERVER_ADDRESS}:3001
+      - REKONI_URL=http://${SERVER_ADDRESS}:4004
+      - CALENDAR_URL=http://${SERVER_ADDRESS}:8095
+      - GMAIL_URL=http://${SERVER_ADDRESS}:8088
+      - TELEGRAM_URL=http://${SERVER_ADDRESS}:8086
+      - UPLOAD_URL=/files
+      - ELASTIC_URL=http://elastic:9200
+      - COLLABORATOR_URL=ws://${SERVER_ADDRESS}:3078
+      - COLLABORATOR_API_URL=http://${SERVER_ADDRESS}:3078
+      - MINIO_ENDPOINT=minio
+      - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+      - MONGO_URL=mongodb://mongodb:27017
+      - TITLE=Huly Self Hosted
+      - DEFAULT_LANGUAGE=en
+      - LAST_NAME_FIRST=true
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  collaborator:
+    image: hardcoreeng/collaborator:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+      - transactor
+    ports:
+      - 3078:3078
+    environment:
+      - COLLABORATOR_PORT=3078
+      - SECRET=${SERVER_SECRET}
+      - ACCOUNTS_URL=http://account:3001
+      - UPLOAD_URL=/files
+      - MONGO_URL=mongodb://mongodb:27017
+      - MINIO_ENDPOINT=minio
+      - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  transactor:
+    image: hardcoreeng/transactor:${HULY_VERSION}
+    links:
+      - mongodb
+      - elastic
+      - minio
+      - rekoni
+      - account
+    ports:
+      - 3333:3333
+    environment:
+      - SERVER_PORT=3333
+      - SERVER_SECRET=${SERVER_SECRET}
+      - SERVER_CURSOR_MAXTIMEMS=30000
+      - ELASTIC_URL=http://elastic:9200
+      - ELASTIC_INDEX_NAME=huly_storage_index
+      - MONGO_URL=mongodb://mongodb:27017
+      - METRICS_CONSOLE=false
+      - METRICS_FILE=metrics.txt
+      - MINIO_ENDPOINT=minio
+      - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+      - REKONI_URL=http://rekoni:4004
+      - FRONT_URL=http://${SERVER_ADDRESS}:8087
+      - SERVER_PROVIDER=ws
+      - ACCOUNTS_URL=http://account:3001
+      - LAST_NAME_FIRST=true
+      - UPLOAD_URL=http://${SERVER_ADDRESS}/files
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  rekoni:
+    image: hardcoreeng/rekoni-service:${HULY_VERSION}
+    ports:
+      - 4004:4004
+    environment:
+      - SECRET=${SERVER_SECRET}
+    deploy:
+      resources:
+        limits:
+          memory: 500M
+    restart: unless-stopped
+    networks:
+      - dokploy-network
```

Comment:
is not necessary to add the dokploy network, when you assign a domain to a service it automatically will add this network